org_status/org_hosts/github.py

```diff
@@ -56,8 +58,12 @@ def process_repository(self, repo, branch='master'):
         elif Status.ERROR in repo_status:
             repo_status = Status.ERROR
 
-        return RepoStatus(repo.web_url, repo_status)
+        return RepoStatus(web_url, repo_status)
 
     @property
     def repositories(self):
-        return self._org.repositories
+        return [repo.web_url for repo in self._org.repositories]
```

Comment:
Please notice that if `sync` is `False`, this method will fail. We need an alternate build process for these classes.